scripts/.eslintrc

```diff
@@ -5,4 +5,7 @@
   "rules": {
     "no-console": "off",
   },
+  "parserOptions": {
+    "ecmaVersion": 2018
+  }
```

Comment:
I'm using some new ES features in the `generate-change-list.js` module. This enables ESLint to parse them.

Reply:
This makes me want to confirm that our minimum version of Node is 6.3+ (that's what's given in [our current docs](https://h-client.readthedocs.io/en/latest/developers/developing/) ). I mean, for generators we're fine. Just curious if we've revisited that version requirement lately.

Reply:
For the rest of the code Node v6.3+ probably still works. This code requires Node 10, which is fine, because it only needs to run on our Jenkins system which uses Node 10 from a Docker container.